src/wrapper/jnienv.rs

```diff
@@ -169,6 +169,29 @@ impl<'a> JNIEnv<'a> {
         )
     }
 
+    /// Tests whether an object is an instance of a class.
```

Comment:
I'd consider using the wording from the docs because there is a significant difference between this method & `instanceof`: Returns true if the object reference can be cast to the given type.